openmicroscopy/bioformats
# Fix writing of tiff description (rebased onto dev_5_0)

This is the same as gh-1459 but rebased onto dev_5_0.

----

To test the PR

* Drop bio-formats_tools into imageJ plugins
* Import an image e.g. png (This will generate some size with microns)
* export the image as OME-TIFF
* Check that the OME-TIFF is opened using the OMETiff reader either by using showinf or BF-importer in imageJ

To come: test but this PR should allow @bramalingam and @pwalczysko to review gh-1442

You will need to edit the description of the OME-TIFF and add a unicode character before import
